### PR TITLE
Us01 sign up form

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Gumatan — DISMISSED** [review]

**Review by Gumatan — DISMISSED** [review]

**Review by Gumatan — DISMISSED** [review]

**Review by Gumatan — DISMISSED** [review]

File: frontend/src/components/Formulaire.jsx [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    this.state = {
+      user: {
+        lastname: "",
+        firstname: "",
+        age: "",
+        gender: "",
+        pseudo: "",
+        email: "",
+        password: "",
+        pays: "",
```
Comment (LoicBrassart) [inline frontend/src/components/Formulaire.jsx]: `country`

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
- To me your components should be named in English. (Formulaire.jsx)
- Be careful around your field names ;-)

Should be a quick fix overall

## Notes
I don't really like how you implemented Formulaire.jsx, even though you did it well. We never saw something similar so that's just an advice, but I think I'd have created sub-components (in the same file) to handle every page, and called them in the `render()` of ther main one. Not so sure though, so I'll let it slide

**Review by Gumatan — DISMISSED** [review]

**Review by Gumatan — DISMISSED** [review]

**Review by Gumatan — DISMISSED** [review]

File: frontend/src/components/GameList.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
+const GameList = [
```
Comment (LoicBrassart) [inline frontend/src/components/GameList.jsx]: That's not a component, it has nothing to do in this folder

File: frontend/src/components/ScrollButton.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+/*export default function scrollWin() {
```
Comment (LoicBrassart) [inline frontend/src/components/ScrollButton.jsx]: Dead code

File: frontend/src/components/style/GameChoice.scss [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.gamePage {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 250vw;
+  height: 40vh;
+  //overflow-x: scroll;
```
Comment (LoicBrassart) [inline frontend/src/components/style/GameChoice.scss]: Dead code

File: frontend/src/components/GameChoice.jsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "./style/GameChoice.scss";
+
+
+const name = 'League of legends';
```
Comment (LoicBrassart) [inline frontend/src/components/GameChoice.jsx]: These variables are never called in your file, remove them

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some things to change, my man!

**Review by Gumatan — DISMISSED** [review]

File: frontend/package.json [hunk 6]
```diff
@@ -7,6 +7,9 @@
     "react": "^16.12.0",
     "react-animated-slider": "^2.0.0",
     "react-dom": "^16.12.0",
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
+    "react-slick": "^0.25.2",
```
Comment (Gumatan) [inline frontend/package.json]: react-slick, ce package n'est appelé nul part.

File: frontend/src/App.js [hunk 7]
```diff
@@ -1,7 +1,21 @@
...
-  return <div></div>;
+
+
+function App() {
+  return (
+    <div className="App">
+      <div className="bigGamePage">
+      <div className="gamePage">
+      {GameList.map ((game) => {
+      return (<GameChoice name={game.name} picture={game.picture} click={game.click} />)})}
```
Comment (Gumatan) [inline frontend/src/App.js]: le props click est déclaré mais n'est pas utilisé dans le composant

File: frontend/src/GameList.jsx [hunk 8]
```diff
@@ -0,0 +1,103 @@
+const GameList = [
+  {
+    name: 'League of Legends',
+    picture: '/pictures/league_of_legends.png',
+    click: true
```
Comment (Gumatan) [inline frontend/src/GameList.jsx]: la clé click ne sert nul part dans le projet

File: frontend/src/index.js [hunk 9]
```diff
@@ -1,10 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import "./reset.css";
 import "./index.css";
```
Comment (Gumatan) [inline frontend/src/index.js:4]: le reset css n'est pas appelé dans le projet

**Review by Gumatan — COMMENTED** [review]
merci de régler ces ptit problèmes :) et sa passe.
il serait pas mal de passer ces composants en fonctionnels / hooks par la suite. (non bloquant)

**Review by Gumatan — APPROVED** [review]

File: frontend/src/components/GameChoice.jsx [hunk 10]
```diff
@@ -14,16 +11,16 @@ class GameChoice extends React.Component {
...
-        return  ( 
+        return  (
             <div className="gameChoice"
                 onClick={event => {
                     const newClick = !this.state.click;
                     this.setState({click: newClick});
                 }}> 
-                    <button className={this.state.click ? "status-on" : "status-off"}>
-                        <img className="picture" src={this.props.picture} /></button>
+                <button className={this.state.click ? "status-on" : "status-off"}>
```
Comment (LoicBrassart) [inline frontend/src/components/GameChoice.jsx:20]: You removed the "click" key but use it here, don't you ?

**Review by LoicBrassart — APPROVED** [review]
(GameList is not a component, so it should be a `.js` file)

There is a potential problem with the "click" key. I'll let you decide if you want to merge this PR and fix it later

**Pull request merged**